webui/react/src/e2e/models/BasePage.ts

```diff
@@ -27,6 +33,23 @@ export abstract class BasePage implements ModelBasics {
   get pwLocator(): Locator {
     return this._page.locator(':root');
   }
+  /**
+   * The title of the page. Format of [prefix - ]appTitle
+   * @param prefix
+   */
+  public static getTitle(prefix: string = ''): string {
```

Comment:
i think i might prefer `get title()` similar to how we use `get root`. instead of taking `prefix` as an arg, we should use `this._title`, which would be defined as `abstract readonly _title` in `basePage`. or maybe `abstract readonly _baseTitle` might be more clear?